JSDemos/Demos/Toolbar/Adaptability/React/App.tsx

```diff
@@ -149,7 +149,7 @@ function App() {
   const renderTextAlignMenu = React.useCallback(
     () => (
       <ButtonGroup
-        displayExpr="text"
```

Comment:
Check this line
Our docs and typescript say that there is no displayExpr in ButtonGroup